community-operators/rook/rook.v0.1.0.csv.yaml

```diff
@@ -0,0 +1,316 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  creationTimestamp: null
+  name: rook.v0.1.0
+  namespace: placeholder
+  annotations:
```

Comment:
Additional annotations fields used in operatorhub.io can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md
Namely, the following fields are missing:
- `metadata.annotations.description`: (required) short one-line description of the operator (no markdown)
- `metadata.annotations.categories`: a category that your application falls under (values listed in the doc above)
- `metadata.annotations.capabilities`: (required) support/maturity level of the operator (values listed in doc above)
- `metadata.annotations.repository`: source code repository for your operator as a url
- `metadata.annotations.createdAt`: data and time operator image was created
- `metadata.annotations.containerImage`: (required) operator image link. Should match the image used for the operator deployment, tag and all
- `spec.icon`: an icon for your operator stored as base64
Be sure to update both community-operators and upstream-community-operators versions of the CSV. Let me know if you have any questions about these fields.